src/app/page.js

```diff
@@ -2,15 +2,17 @@ import Landing from "@/components/home/Landing";
 import Goal from "@/components/home/Goal";
 import Involve from "@/components/home/Involve";
 import Upcoming from "@/components/home/Upcoming";
+import Button from "@/components/Button";
 
 const Home = () => {
   return (
-    <div>
+    <header>
       <Landing />
       <Goal />
       <Involve />
       <Upcoming />
-    </div>
+      <Button color="blue" text="Sign Up" />
```

Comment:
if the problem you are having with your local version of the website is getting the button to show up, it's probably because there's an error in the component file(noted in other comment), if the issue is something else please let me know and I'll look into it.